src/chi/components/modal/modal.scss

```diff
@@ -114,6 +114,16 @@ $animation-duration: 0.5s;
     }
   }
 
+  &__wrapper {
```

Comment:
This solves the problem perfectly but breaks the positioning of existing modals by rendering them in the top left of the screen. Can we come up with a solution that horizontally centers existing modals that aren't wrapped in the new div?